modules/core/client/directives/title.client.directives.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+angular.module('core').directive('updateTitle', ['$rootScope', '$timeout', '$interpolate', '$state',
```

Comment:
Can you come up with a better name for the directive? Also, whatever name you land on, match the name of the file with it.